src/app/modules/users/users-list/users-list.component.ts

```diff
@@ -95,6 +105,39 @@ export class UsersListComponent implements OnInit {
     this.dataSource.filter = filterValue;
   }
 
+  // To Download the csv file
+  downloadFile(data: any) {
```

Comment:
Please use file saver plugin for saving file

Reply:
changed